test/e2e/framework/framework.go

```diff
@@ -635,6 +635,11 @@ func KubeDescribe(text string, body func()) bool {
 	return ginkgo.Describe("[k8s.io] "+text, body)
 }
 
+// SIGDescribe is a wrapper function for ginkgo describe. Adds SIG namespacing.
+func SIGDescribe(sig, text string, body func()) bool {
+	return ginkgo.Describe("[sig-"+sig+"] "+text, body)
+}
+
```

Comment:
Added this following review suggestion from https://github.com/kubernetes/kubernetes/pull/98957 
cc: @liggitt @wojtek-t